clickhouse-jdbc/src/test/java/com/clickhouse/jdbc/ClickHousePreparedStatementTest.java

```diff
@@ -2005,6 +2007,53 @@ public void testInsertWithSettings() throws SQLException {
         }
     }
 
+    @Test(groups = "integration")
+    public void testGetMetaData() throws SQLException {
+        try (Connection conn = newConnection(new Properties());
+                PreparedStatement ps = conn.prepareStatement("select ? a, ? b")) {
+            ResultSetMetaData md = ps.getMetaData();
```

Comment:
What happens if we pass DDL (e.g. CREATE TABLE) to `conn.prepareStatement`? It would be nice to add a test in my opinion.

Reply:
Thanks for taking your time to review the PR!

> What happens if we pass DDL (e.g. CREATE TABLE) to conn.prepareStatement?

`null` will be returned at this point. Below is the check in `SqlBasedPreparedStatement.java`. However, in ClickHouse, create table on cluster will actually return ResultSet, but I guess we don't need that as it's not helping for push-down implementation.
```java
if (parsedStmt.isRecognized() && !parsedStmt.isQuery()) {
    return null;
}
```

> It would be nice to add a test in my opinion.

Yes, the implementation is incomplete and I'll add more tests for sure.